action-templates/actions/action-checkout/action.yml

```diff
@@ -1,8 +1,28 @@
 name: "Checkout Code"
 description: "A wrapper for actions/checkout with no args"
```

Comment:
this isn't correctt as you added now a filter.

Reply:
```suggestion
description: "A wrapper for actions/checkout  and paths filter with input of filters"
```